posts/2021-12-15-schema-evolution.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Schema Evolution"
+description: "Exploring the ability to update the data model in a Universal Application (Uni)"
+date: "2021-12-15"
+categories:
+  - announcements
+authors:
+  - Brian McNamara
+---
+
+Vendia Share allows users to share a single source of truth in a secure and scalable manner through Universal Applications (Unis). Unis are automatically created using a user-supplied data model. Vendia Share takes the user supplied data model and deploys a Uni in a matter of minutes.  This includes GraphQL APIs to interact with the Uni and a serverless distributed ledger that keeps data consistent across all participants - all managed as a service.
+
+As Uni participants grow and as the fidelity of solutions increase over time, it's common for users to want to evolve the original data model used to create the Uni.  Historically, there was no way for users to change the data model once a Uni was created. Any data model change to a Uni required destroying it and its accompanying nodes and recreating it with the new model. 
```

Comment:
```suggestion
As Uni participants grow and as the fidelity of solutions increase over time, it's common for users to want to revise the original data model used to create the Uni.  Historically, there was no way for users to change the data model once a Uni was created. Any data model change to a Uni required destroying it and its accompanying nodes and recreating it with the new model. 
```

Reply:
I know we call it evolution, but that term doesn't resonant with me. It could be because I don't know the space well and that's the right term. But to me revise makes more sense than evolve.

Reply:
It may be a semantic thing but I equate evolve with grow. This is important as schema evolution doesn't allow for any revision - only those that are more additive in nature.